tools/dev/loki-boltdb-storage-s3/config/loki.yaml

```diff
@@ -67,7 +67,6 @@ ingester_client:
     remote_timeout: 1s
 limits_config:
     cardinality_limit: 100000
-    enforce_metric_name: false
```

Comment:
@jeschkies can we revert that?

Reply:
I've you want to. I need to add a CI job for this 😊